### DISCUSSION TITLE
Keep the current file name and collapse buttons in view

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#51259] [Mobile] [answered]
GitHub Mobile beta

1. Visit https://github.com/cert-manager/example-approver-policy-plugin/pull/1
2. View the code
3. Scroll down
    ![Screenshot_20230328-142016.png](https://user-images.githubusercontent.com/2119212/228332363-803d9228-f3e2-4248-9d0d-912b437ff729.png)

It's a pain to scroll back up to get to the collapse widget, and it also means I don't always know which file I'm looking at.

It doesn't have to be at the very top of there's a previous file also on screen, but when the previous file has scrolled off, it should snap the current file and checkbox to the top and let the file contents scroll off without losing the filename/checkbox.

In this case, the previous file, whose name you don't know because it's scrolled off screen should have been snapped to the top until the next one replaced it.

**Comment by iskae** [comment, marked as answer]
Hello @jsoref 👋 

Thanks a bunch for reaching out!

Yep, we have been discussing replacing this with sticky headers lately too ❤️ I have just created an internal issue for tracking purposes and hopefully we can tackle this soon 😊 

Please stay tuned for the future versions!

Cheers!
